release/readme.html

```diff
@@ -328,18 +450,14 @@ <h2><a id="Platforms_90"></a>Platforms</h2>
     <h2><a id="Installation_contents_100"></a>Installation contents</h2>
     <p>The installation creates a program folder with icons/shortcuts for the programs/files in EnergyPlus that can be
         accessed from Windows menus, including removing EnergyPlus from your system. In Windows OS, these will be
-        available
-        from the “Programs” area in the Start Menu. Note that only a few of the programs will store anything in your
-        Windows\System directory (IDFEditor, EPLaunch, WeatherConverter).</p>
+        available from the “Programs” area in the Start Menu. Note that only a few of the programs will store anything
+        in your Windows\System directory (IDFEditor, EPLaunch, WeatherConverter).</p>
     <h2><a id="Known_problems_104"></a>Known problems</h2>
     <p>While every effort has been made to clean up all the “defects” that have occurred during our testing, quite a few
         known (and even more unknown) probably remain. Specifics on the remaining known problems can be found on the
-        GitHub
-        repository <a href="https://github.com/NREL/EnergyPlus/issues">issues page</a>. If you are interested in what
-        has
-        been
-        addressed, the issues resolved in this release are installed similarly in the <a
-                href="changelog.html">changelog</a>.
+        GitHub repository <a href="https://github.com/NREL/EnergyPlus/issues">issues page</a>. If you are interested in
+        what has been addressed, the issues resolved in this release are installed similarly in the
+        <a href="changelog.html">changelog</a>.
```

Comment:
Seems like the changelog should be more prominent?

Reply:
Made a standalone line in the highlight list of changes to inform the user of the changelog.